### DISCUSSION TITLE
Alert with PostgreSQL Integration after inadvertently clicking on Integrations

### DISCUSSION THREAD

**Discussion by krivulcik** [ravendb/ravendb#14157] [Q&A] [answered]
We have unintentionally clicked on the "Integrations" link for a database on one node in a cluster.

Now, after every visit, we get the following alert:
![image](https://user-images.githubusercontent.com/7997055/166469815-5333d914-314d-4820-917e-c1bc4886b6f9.png)

Clicking the link again yields the following alert:
![image](https://user-images.githubusercontent.com/7997055/166469936-82c0a365-1973-4a5a-a0c8-1b869c625742.png)

Is it possible to get rid of this alert?

Other nodes in the same cluster don't show this alert.

The cluster has a Professional license, and the PostgreSQL integration is not enabled in the configuration file (Integrations.PostgreSQL.Enabled is not set). Experimental features are not enabled.

**Comment by ayende** [comment, marked as answer]
Are you seeing that when you visit the integration page, or in general on that node?

That generates an alert, which you should be able to dismiss, see:
![image](https://user-images.githubusercontent.com/116915/166633638-cd4b3c2b-db27-4d74-8146-6ecb899dbf46.png)

**Reply by ml054** [reply]
@krivulcik you can track progress here: https://issues.hibernatingrhinos.com/issue/RavenDB-18438

**Reply by krivulcik** [reply]
Thanks, guys, that makes sense and is also not obvious :-).
